dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/SourceCodeRepoInterface.java

```diff
@@ -208,6 +208,7 @@ Entry updateEntryMetadata(Entry entry, AbstractEntryClient.Type type) {
             return entry;
         }
 
+        // for tools if a default version is set, this actually returns the tag name and not the tag repository
```

Comment:
I think this needs elaboration "for tools if a default version is set, this actually returns the tag name in the Docker registry and not the reference in the source code repo"